clients/sail_ui/lib/rpcs/bitwindow_api.dart

```diff
@@ -320,6 +332,190 @@ class _BitwindowAPILive implements BitwindowAPI {
       DeleteAddressBookEntryRequest()..id = id,
     );
   }
+
+  @override
```

Comment:
I would prefer if storeContent, retrieveContent and hasOpReturn  did not get added to the API. `storeContent` and `retrieveContent` fits in the provider.

`hasOpReturn` can be added as a function to NewsProvider() for example. That has already fetched the op returns, and it could just do a simple search